backend/shoulder/s2s/db_models/event_suggestion.py

```diff
@@ -0,0 +1,208 @@
+
+    dist_within_1mi = models.BooleanField()
+    dist_within_5mi = models.BooleanField()
+    dist_within_10mi = models.BooleanField()
+    dist_within_15mi = models.BooleanField()
+    dist_within_20mi = models.BooleanField()
+    dist_within_30mi = models.BooleanField()
+    dist_within_40mi = models.BooleanField()
+    dist_within_50mi = models.BooleanField()
+
+    num_particip_1to5 = models.BooleanField()
+    num_particip_5to10 = models.BooleanField()
+    num_particip_10to15 = models.BooleanField()
+    num_particip_15p = models.BooleanField()
+
+    moday_early_morning = models.BooleanField()
+    monday_morning = models.BooleanField()
+    monday_afternoon = models.BooleanField()
+    monday_everning = models.BooleanField()
+    monday_night = models.BooleanField()
+    monday_late_night = models.BooleanField()
+    monday_early_morning = models.BooleanField()
+    tuesday_early_morning = models.BooleanField()
+    tuesday_morning = models.BooleanField()
+    tuesday_afternoon = models.BooleanField()
+    tuesday_evening = models.BooleanField()
+    tuesday_night = models.BooleanField()
+    tuesday_late_night = models.BooleanField()
+    wednesday_early_morning = models.BooleanField()
+    wednesday_morning = models.BooleanField()
+    wednesday_afternoon = models.BooleanField()
+    wednesday_evening = models.BooleanField()
+    wednesday_night = models.BooleanField()
+    wednesday_late_night = models.BooleanField()
+    thursday_early_morning = models.BooleanField()
+    thursday_morning = models.BooleanField()
+    thursday_afternoon = models.BooleanField()
+    thursday_evening = models.BooleanField()
+    thursday_night = models.BooleanField()
+    thursday_late_night = models.BooleanField()
+    friday_early_morning = models.BooleanField()
+    friday_morning = models.BooleanField()
+    friday_afternoon = models.BooleanField()
+    friday_evening = models.BooleanField()
+    friday_night = models.BooleanField()
+    friday_late_night = models.BooleanField()
+    saturday_early_morning = models.BooleanField()
+    saturday_morning = models.BooleanField()
+    saturday_afternoon = models.BooleanField()
+    saturday_evening = models.BooleanField()
+    saturday_night = models.BooleanField()
+    saturday_late_night = models.BooleanField()
+    sunday_early_morning = models.BooleanField()
+    sunday_morning = models.BooleanField()
+    sunday_afternoon = models.BooleanField()
+    sunday_evening = models.BooleanField()
+    sunday_night = models.BooleanField()
+    sunday_late_night = models.BooleanField()
+
+    duration_1hr = models.BooleanField()
+    duration_2hr = models.BooleanField()
+    duration_3hr = models.BooleanField()
+    duration_4hr = models.BooleanField()
+    duration_5hr = models.BooleanField()
+    duration_6hr = models.BooleanField()
+    duration_7hr = models.BooleanField()
+    duration_8hr = models.BooleanField()
+    duration_9hr = models.BooleanField()
```

Comment:
since max duration is set to 8hrs, we probably don't need the 9-12 hour columns anymore